cmd/openshift-install/destroy.go

```diff
@@ -39,32 +40,44 @@ func newDestroyCmd() *cobra.Command {
 }
 
 func newDestroyClusterCmd() *cobra.Command {
-	return &cobra.Command{
+	var reportQuota bool
+	cmd := &cobra.Command{
 		Use:   "cluster",
 		Short: "Destroy an OpenShift cluster",
 		Args:  cobra.ExactArgs(0),
 		Run: func(_ *cobra.Command, _ []string) {
 			cleanup := setupFileHook(rootOpts.dir)
 			defer cleanup()
 
-			err := runDestroyCmd(rootOpts.dir)
+			err := runDestroyCmd(rootOpts.dir, reportQuota)
 			if err != nil {
 				logrus.Fatal(err)
 			}
 		},
 	}
+	quotaFlag := "report-quota-footprint"
+	cmd.Flags().BoolVar(&reportQuota, quotaFlag, false, "Report the quota footprint that the cluster had.")
+	_ = cmd.Flags().MarkHidden(quotaFlag) // only error possible is the flag does not exist, but we just added it
+	return cmd
```

Comment:
I'm reluctant to even bring this up due to risk of bike shedding, but did we discuss hiding this behind an environment variable? Environment variables seem to be the preferred way of changing behavior for CI, and we don't frequently use flags.

That said, I'm totally fine with this and the fact that it is hidden would allow us to make subsequent changes if we need.

Reply:
I have no preference, whichever your preference I will implement.